src/main/java/org/folio/rest/impl/FiscalYearsApi.java

```diff
@@ -22,13 +22,19 @@
 public class FiscalYearsApi implements FinanceFiscalYears {
 
   private static final String FISCAL_YEARS_LOCATION_PREFIX = getEndpoint(FinanceFiscalYears.class) + "/%s";
+  private static final int FISCAL_YEAR_LENGTH = 4;
 
   @Validate
   @Override
-  public void postFinanceFiscalYears(String lang, FiscalYear entity, Map<String, String> headers,
+  public void postFinanceFiscalYears(String lang, FiscalYear fiscalYear, Map<String, String> headers,
       Handler<AsyncResult<Response>> handler, Context ctx) {
     FiscalYearsHelper helper = new FiscalYearsHelper(headers, ctx, lang);
-    helper.createFiscalYear(entity)
+    String code = fiscalYear.getCode();
+
+    // series should always be calculated
+    fiscalYear.setSeries(code.substring(0, code.length() - FISCAL_YEAR_LENGTH));
```

Comment:
This lines of code repeated 2 times in POST method and PUT method. Wouldn't it be better to move this part of code into separate method something like
```
private FiscalYear getFYearWithSeries(FiscalYear fiscalYear){
   String code = fiscalYearRequest.getCode();
   return fiscalYear.withSeries(code.substring(0, code.length() - FISCAL_YEAR_LENGTH));
}
```